index.html

```diff
@@ -43,11 +43,108 @@ <h1 class="prof">I am Software Developer</h1>
             </a>
-        
-
+        </div>
+    </section>
 
+     <!-- content 2 -->
+    <section class="main2">
+        <div class="title">
+            <h1>My Recent Work</h1>
+            <div class="end-line"></div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>HTML5</li>
+                    <li>HTML5</li>
+                    <li>HTML5</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul >
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
         </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>HTML5</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button">See project</button>
+            </div>
+        </div>
+    </section>
+    <!-- content 2 -->
+    <section class="main3">
+        <h1>About me</h1>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+        <button type="button">Get my Resume</button>
     </section>
-  
+
+<section class="card-info">
+    <div class="card">
+        <img src="./image/icon-languages.svg" alt="">
+        <h1>Language</h1>
+        <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>html</li>
+            <li>css</li>
+        </ul>
+    </div>
+    <div class="card">
+        <img src="./image/icon-frameworks.svg" alt="">
+        <h1>Framework</h1>
+        <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>Rspec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
```

Comment:
- [ ] Could you please fix this spacing and make it match the Figma design 

Yours | Figma Design
:--:|:--:
![image](https://user-images.githubusercontent.com/87689547/148622841-5a607c7a-8f44-48e4-bf58-f187148c124f.png) | ![image](https://user-images.githubusercontent.com/87689547/148622862-565a0cae-ecd3-486e-9fa3-6311de26fed0.png)